README.md

```diff
@@ -96,18 +96,22 @@ or 365 days for users without a password (machine users).
 It also disables permanent users who have left the Guardian.
 
 The reaper sends email notifications to the AWS account the user is in, before disabling a user.
-The emails are sent via Anghammarad and uses it's AWS Account to email address mappings.
+The emails are sent via Anghammarad and uses its AWS Account to email address mappings.
 
 You can also find the dynamo table in the Security AWS Account.
 
 ### Lambda
 Security HQ holds a Lambda, which checks for security groups that are open to the world, except ELB groups. This data is used for `https://security-hq.gutools.co.uk/security-groups`.
 
 It is deployed as a stack set and is defined in  `cloudformation/watched-account.template.yaml`.
-This lambda is deployed manually by creating a JAR file locally and uploading it to S3: `s3://guardian-dist/guardian/PROD/securitygroups-lambda/`. The version name is important, because
-the cloudformation has a paramter, `version`, which is used to locate the correct S3 file. The stackset is then deployed manually again to all accounts from the root account.
 
-The build and deploy process is manual at present. The lambda was deployed once in 2018 and hasn't been updated since. Health tickets have been put onto the backlog to try to improve this process.
+#### Deployment
```

Comment:
Worth explicitly noting that the process is currently manual?

Reply:
done